content/coronavirus_landing_page.yml

```diff
@@ -196,13 +196,17 @@ content:
           list:
             - label: NHS guidance for people working in healthcare
               url: https://www.england.nhs.uk/coronavirus/
-            - label: Guidance on using appropriate personal protective equipment (PPE)
+            - label: Using PPE (personal protective equipment)
               url: /government/collections/coronavirus-covid-19-personal-protective-equipment-ppe
-            - label: Guidance on adult social care
+            - label: Managing possible or confirmed cases
+              url: /government/collections/wuhan-novel-coronavirus#guidance-for-the-management-of-possible-or-confirmed-cases
```

Comment:
Do you think we need to anchor link here? it's less accessible and because it's the first section?